apps/test/unit/templates/MultiCheckboxSelectorTest.js

```diff
@@ -107,4 +107,30 @@ describe('MultiCheckboxSelector', () => {
       expect(onChange.lastCall.args[0]).to.deep.equal([]);
     });
   });
+  describe('no header', () => {
+    beforeEach(() => {
+      render(
+        <MultiCheckboxSelector
+          noHeader={true}
+          header="Some Items"
+          items={['one', 'two', 'three']}
+          selected={['two']}
+          onChange={onChange}
+        >
+          <ItemComponent />
+        </MultiCheckboxSelector>
+      );
+    });
+
+    it('should not render a header', () => {
+      expect(
+        header.matchesElement(
+          <h2>
+            <input type="checkbox" checked={false} />
+            Some Items
```

Comment:
nice! this format seems way better than writing code to inspect the result of what gets rendered.